polaris-core/src/main/java/org/apache/polaris/core/secrets/UnsafeInMemorySecretsManager.java

```diff
@@ -45,6 +43,8 @@ public class UnsafeInMemorySecretsManager implements UserSecretsManager {
   private static final String CIPHERTEXT_HASH = "ciphertext-hash";
   private static final String ENCRYPTION_KEY = "encryption-key";
 
+  private static final String SECRET_MANAGER_TYPE = "unsafe-in-memory";
```

Comment:
Can we make a static constant like `UnsafeInMemorySecretsManager.prefix`? (NID?)